### DISCUSSION TITLE
Welcome to ColorPickerView Discussions!

### DISCUSSION THREAD

**Discussion by skydoves** [skydoves/ColorPickerView#63] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by trungdn0812** [comment]
Tks bro, your library help me a lot. And I got an error "Cannot find local variable args" when I used ImageDecoder.decodeDrawable to create Drawable. My app requires color correctly very high. So it has to always check the color spaces and does conversation base on its. 
So can you help me fix it or explant to me how does your library convert color spaces into HSV?
I'm new.
Tks a lot!
Here some information when I debug my app

"Cannot find local variable args"
void dumpInner(@NonNull String prefix, @Nullable FileDescriptor fd,
            @NonNull PrintWriter writer, @Nullable String[] args) {
        if (args != null && args.length > 0) {
            // Handle special cases
            switch (args[0]) {
                case "--autofill":
                    dumpAutofillManager(prefix, writer);
                    return;
                case "--contentcapture":
                    dumpContentCaptureManager(prefix, writer);
                    return;
            }
        }
